contrib/docker/docker-compose.yml

```diff
@@ -6,7 +6,7 @@ version: '3'
 services:
 
   synapse:
-    image: docker.io/matrixdotorg/synapse:latest
+    image: matrixdotorg/synapse:develop
```

Comment:
wouldn't it be better to use the master branch instead?

Reply:
This should definitely not be develop. About using `latest`, take a look at [my comment on #3407 ](https://github.com/matrix-org/synapse/issues/3407#issuecomment-406224633)

Reply:
Also, while it is possible to remove the `docker.io/` prefix, I'd prefer to keep it, to make sure that it pulls from the registry we expect it to pull from. While many systems try docker.io first, that is not true for all systems.

Reply:
@jcgruenhage: I've read your comment on #3407 and I'm still confused. should we accept this PR? Reject it? should we ask for it to be changed in a specific way?

Reply:
I'd say we should reject this PR and instead clarify our documentation. The latest tag that is currently in the docker-compose file and in the documentation are for when people build the images themselves.